app/src/main/java/org/listenbrainz/android/viewmodel/SearchViewModel.kt

```diff
@@ -0,0 +1,44 @@
+package org.listenbrainz.android.viewmodel
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+import org.listenbrainz.android.model.SearchResult
+import org.listenbrainz.android.repository.SocialRepository
+import org.listenbrainz.android.util.Resource
+import javax.inject.Inject
+
+@HiltViewModel
+class SearchViewModel @Inject constructor(private val repository: SocialRepository) : ViewModel() {
+    
+    private val _searchState = MutableStateFlow<Resource<SearchResult>>(Resource.loading())
+    val searchState = _searchState.asStateFlow()
+    
+    private val _test = MutableStateFlow<List<String>>(emptyList())
+    val test = _test.asStateFlow()
+    
+    init {
+        searchUser("Jasjeet")
```

Comment:
As long as this won't be hardcoded in future, I'm fine with it :)

Reply:
Yeah this is just for testing, its already removed on local.